python/paddle/fluid/contrib/slim/quantization/qat2_int8_mkldnn_pass.py

```diff
@@ -333,6 +333,7 @@ def _optimize_fp32_graph(self, graph):
                                      'conv_elementwise_add_mkldnn_fuse_pass')
             graph = self._apply_pass(graph, 'conv_relu_mkldnn_fuse_pass')
             graph = self._apply_pass(graph, 'conv_relu6_mkldnn_fuse_pass')
+            graph = self._apply_pass(graph, 'scale_matmul_fuse_pass')
```

Comment:
Does it have to be inside this `if self._is_conv_quantized():` ?

Reply:
Done